vllm/config.py

```diff
@@ -971,7 +971,8 @@ def __init__(self,
                  is_multimodal_model: bool = False,
                  preemption_mode: Optional[str] = None,
                  num_scheduler_steps: int = 1,
-                 send_delta_data: bool = False) -> None:
+                 send_delta_data: bool = False,
+                 policy: str = "sp") -> None:
```

Comment:
shouldn't this be "priority"?

Reply:
Yes, changed to priority. The earlier use of "sp" was for strict priority. Based on the design discussion, the priority policy is general enough.